### ISSUE TITLE
LED's not flashing green

### ISSUE THREAD

**Issue by capsulecorplab** [tetrabiodistributed/PAPRA-PCB#54]
This may be a non-issue, since it happened after using it to power a stepper motor driver for a NEMA17, but the LED arrays don't flash green when the control knob is turned back on and instead goes directly to red. The battery seems fine and I don't see any burnt IC's. Any thoughts?
![PXL_20210702_184704664](https://user-images.githubusercontent.com/14095576/124322589-ee738180-db3c-11eb-970e-d8e81ed19c8f.jpg)
![PXL_20210702_184750503](https://user-images.githubusercontent.com/14095576/124322595-f3d0cc00-db3c-11eb-9222-c5598fb71fa6.jpg)
![PXL_20210702_184957285](https://user-images.githubusercontent.com/14095576/124322602-f6cbbc80-db3c-11eb-95c7-358de259ada3.jpg)
![PXL_20210702_201339188](https://user-images.githubusercontent.com/14095576/124324179-c76a7f00-db3f-11eb-8f61-6c90fb16a5b7.jpg)

**Comment by mmroden** [comment]
Iirc the LEDs are all one color, not multiple colors for different states.  Feature request, maybe?

**Comment by capsulecorplab** [comment]
@mmroden you're right, the LED color was red. but this issues is concerning the LED arrays that indicate the battery level that aren't coming on

**Comment by mechatroniks-git** [comment]
@capsulecorplab - v.2 had a critical failure mode that was fixed later. Issue was if the output fan was shorted, the microcontroller would blow up. 

If you connected a low impedance load (NEMA stepper motor) to the FAN output, this would likely cause the chip failure.

**Comment by mmroden** [comment]
@mechatroniks-git will get @capsulecorplab a replacement
